doc/website/getting-started/overview.md

```diff
@@ -11,637 +11,220 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
-Notice that ``sample.get()`` returns a ``void*`` to the memory ``chunk`` were we can then place our data.
-Depending on what we want to send, we may also use ``memcpy`` to copy the data to the ``chunk``.
-
-```cpp
-if (!result.has_error())
-{
-    auto& sample = result.value();
-    void* chunk = sample.get();
-    new (chunk) CounterTopic(73);
-    sample.publish();
-}
-else
-{
-    // could not acquire chunk, handle the error
-}
-```
+A ``ServiceDescription`` in iceoryx represents the data to be transmitted and is uniquely identified by three string
+identifiers.
 
-Here placement new is required, there is no preconstructed object at ``sample.get()``.
+1. ``Group`` name
+2. ``Instance`` name
+3. ``Topic`` name
 
-##### 2. Functional approach with loaning
+A triple consisting of such strings is called a ``ServiceDescription``. The service model of iceoryx is derived
+from AUTOSAR and is still used in the API with these names. The so called canonical protocol is implemented in the
+namespace ``capro``.
 
-```cpp
-publisher.loan(sizeof(CounterTopic))
-    .and_then([&](iox::popo::Sample<void>& sample) {
-        new (sample.get()) CounterTopic(73);
-        sample.publish();
-    })
-    .or_else([](iox::popo::AllocationError) {
-        /* handle the error */
-    });
-```
+The following table gives an overview over the different terminologies and the current mapping:
 
-Notice that we get an untyped sample, ``iox::popo::Sample<const void>``. We could also use ``auto& sample`` in the
-lambda arguments to shorten it. Again we access the pointer to the underlying raw memory of the sample and
-construct the data we want to send.
+|         | Group   | Instance         | Topic                  |
+|---------|---------|------------------|------------------------|
+| ROS2    | Type    | Namespace::Topic | -                      |
+| AUTOSAR | Service | Instance         | Event                  |
+| DDS     | -       | -                | /Group/Instance/Topic  |
 
-#### Creating a subscriber
+Service and instance are like classes and objects in C++. So you always have a specific instance of a service during
+runtime. The mapping will be reworked with release v2.0.
 
-While the string identifiers still have to match those in the publisher, as in the untyped publisher there is no
-template type argument.
+Two ``ServiceDescription``s are considered matching if all these three strings are element-wise equal, i.e. group,
+instance and topic names are the same for both of them.
 
-```cpp
-iox::popo::UntypedSubscriber subscriber({"Group", "Instance", "CounterTopic"});
-subscriber.subscribe();
-```
+This means the group and instance identifier can be ignored to create different ``ServiceDescription``s. They will be
+needed for advanced filtering functionality in the future.
 
-#### Receiving data
+The data type of the transmitted data can be any C++ class, struct or plain old data type as long as it satisfies the 
+following conditions:
+- no heap is used
```

Comment:
maybe some prominent examples can be mentioned: `std::string` or `std::vector`.
On the other hand we should mention what is possible to use, like arrays filled with fixed-size data.